position_statements.py

```diff
@@ -129,6 +134,10 @@ def expanded_datavalue(datavalue):
     commands.append(command)
 
 for command in commands:
+    summary = 'Edited with PositionStatements'
+    if user_name:
+        summary += ' on behalf of [[User:{}]]'.format('Chris Mytton')
```

Comment:
Shouldn't this be filling in `user_name` rather than 'Chris Mytton'?

Reply:
Oh, erm, yep, that's embarrassing!

Reply:
Fixed in https://github.com/everypolitician/position_statements/pull/10